### PR TITLE
Change SHACL shapes to enforce only one symbol per entity

### LINKED ISSUE #1011: SHACL shapes do not enforce one qudt:symbol per entity

### PR DESCRIPTION
**Pull request by fkleedorfer**
This PR is not ready to merge.

It reveals that several quantitykinds have 2 symbols, we need to fix this before merging

Fixes #1011

### CODE REVIEW

**Comment by fkleedorfer** [comment]
have a look at the failing checks to see which quantitykinds need to be fixed

**Review by fkleedorfer — COMMENTED** [review]
An entry in the changelog would be good

**Comment by steveraysteveray** [comment]
OK, I made my best guesses on what should be the qudt:symbol value, and what should be the qudt:altSymbol value.
This is now ready for review and merge.

File: pom.xml [hunk 1]
```diff
@@ -135,11 +144,9 @@
                                     <!--skip>true</skip-->
                                     <shapes>
                                         <include>target/dist/validation/COLLECTION_QUDT_QA_TESTS_ALL.ttl</include>
-                                        <include>target/dist/schema/shacl/SCHEMA_QUDT_NoOWL.ttl</include>
```
Comment (fkleedorfer) [inline pom.xml]: We lost two lines in pom.xml by copying the pom.xml from the main branch - the lines that actually cause the cardinalities to be checked. 

Rebase would have been the way to go

**Comment by fkleedorfer** [comment]
Good to go now!

**Review by steveraysteveray — APPROVED** [review]

**Pull request merged**